packages/amplify-graphql-transformer-core/src/cdk-compat/stack-synthesizer.ts

```diff
@@ -79,7 +79,7 @@ export class TransformerStackSythesizer extends LegacyStackSynthesizer {
     const s3ObjectUrl = `s3://${bucketName}/${rootKey}/${asset.fileName}`;
 
     return {
-      bucketName, objectKey, httpUrl, s3ObjectUrl, s3Url: httpUrl,
+      bucketName, objectKey, httpUrl, s3ObjectUrl,
```

Comment:
Is this a lint fix ?

Reply:
No. CDK removed that parameter. It was deprecated and is redundant with `httpUrl`.